packages/fx-core/src/plugins/solution/fx-solution/debug/provisionLocal.ts

```diff
@@ -47,6 +47,7 @@ export async function setupLocalDebugSettings(
   const includeBot = ProjectSettingsHelper.includeBot(ctx.projectSetting);
   const includeAAD = ProjectSettingsHelper.includeAAD(ctx.projectSetting);
   const includeSimpleAuth = ProjectSettingsHelper.includeSimpleAuth(ctx.projectSetting);
+  const migrateFromV1 = ProjectSettingsHelper.isMigrateFromV1(ctx.projectSetting);
```

Comment:
Duplicated with `isMigrateFromV1`. Make sure `ProjectSettingsHelper.isMigrateFromV1` only called once.

Reply:
Got it